johnsonandjohnson/Bodiless-JS
# fix(p.sh): Fix env vars are not generated for p.sh.

## Changes
- Add environment vars generation into build step on p.sh.
- Add building docs into build step on p.sh.

## Test Instructions
1. Deploy branch to p.sh and make sure env vars are generated correctly.
   * You can verify that by checking whether you can update the content of a site and it is persisted after the page refresh ( `BODILESS_BACKEND_SAVE_ENABLED='1'` ).
   * Also you can try to open the docs app by clicking the docs icon. ( `BODILESS_DOCS_URL='/___docs'` )

2. Change some values in `examples/test-site/.env.site` and redeploy to p.sh.
3. Make sure env vars on p.sh are changed accordingly.
   * The easiest you can do is to change `BODILESS_DOCS_URL` in `examples/test-site/.env.site` to any`"any_url_string"` and redeploy to p.sh. When you click the docs button it should try to redirect to `/any_url_string`.

## Related Issues
Relates to: https://github.com/johnsonandjohnson/Bodiless-JS/issues/50
